config.toml

```diff
@@ -27,3 +27,29 @@ permissions-bools = [
     "dev-desktop",
     "sync-team-confirmation",
 ]
+
+# GitHub accounts that are allowed to stay in the orgs,
+# even if they may not be members of any team.
+allowed-org-members = [
+    # Bots.
+    "bors",
+    "craterbot",
+    "rust-embedded-bot",
+    "rust-highfive",
+    "rust-lang-core-team-agenda-bot",
+    "rust-lang-glacier-bot",
+    "rust-lang-owner",
+    "rust-timer",
+    "rustbot",
+
+    # Infra admins.
+    "jdno",
```

Comment:
Hmm, we start having a bunch of places where we hardcode infra-admins. Maybe it would be better if we loaded the actual contents of the `infra-admins` team (so that we hardcode only the team in code, but not its members) in code and ignored them?

Reply:
This is a good point, but also, I'm not sure if hardcoding the infra-admins team in the code is the best either. As we are not managing org owners with the team repo for now, I'd skip removing any org owner, and do a manual pass removing inactive org owners.